### ISSUE TITLE
Which document is used for the evaluation of test results? Excuse me?

### ISSUE THREAD

**Issue by Zoeey123** [SilvioGiancola/TrackingNet-devkit#31]

**Comment by Zoeey123** [comment]
metrics.py? What GT_zip for the TrackingNet dataset?

**Comment by SilvioGiancola** [comment]
We provide the code for evaluation in this repo, but do not share the GT to avoid any overfitting on the test set. We do not want participants to train on the GT test set.
